src/routing/http-routing-v1.md

```diff
@@ -130,9 +140,8 @@ Each object in the `Providers` list is a *write provider record*.
 
 - `ProvideResults` is a list of results in the same order as the `Providers` in the request, and the schema of each object is determined by the `Protocol` of the corresponding write object
   - This may contain output information such as TTLs, errors, etc.
-  - It is undefined whether the server will allow partial results
+  - It is undefined whether the server will allow partial results <!-- TODO: maybe add Error field to `announcement` schema and use it in responses ?  -->
```

Comment:
I'd defer specification of partial errors until we have implementations encounter that problem?

Reply:
Yes, tbd. 
This will be something we have to specify if we want to implement `application/x-ndjson` for streaming  PUTs in https://github.com/ipfs/boxo/tree/main/routing/http/client